settings.gradle

```diff
@@ -18,7 +18,7 @@ rootProject.name="clouddriver"
 
 def subs = [
   kato: ["kato-web", "kato-aws", "kato-manual", "kato-jedis", "kato-docker", "kato-cf", "kato-titan", "kato-azure"],
-  oort: ["oort-core", "oort-web", "oort-aws", "oort-gce", "oort-bench", "oort-titan", "oort-cf", "oort-azure"],
+  oort: ["oort-web", "oort-aws", "oort-gce", "oort-bench", "oort-titan", "oort-cf", "oort-azure"],
```

Comment:
I've kept kato-web, mort-web and oort-web so the subprojects they bring in are not added directly into clouddriver-web. Especially for kato-web, it's a handful of dependencies. Thought it was cleaner this way, and can kill off the remaining {kato|mort|oort}-web .gradle files once the subprojects have all been themselves merged into clouddriver.